tutor/specs/components/__snapshots__/sections-filter.spec.jsx.snap

```diff
@@ -12,16 +52,135 @@ exports[`SectionsFilter component renders 1`] = `
     <button
       aria-expanded={false}
       aria-haspopup={true}
-      aria-label="Display sections"
-      className="dropdown-toggle btn btn-default"
+      aria-label="Sections"
+      className="dropdown-toggle btn btn-link"
       disabled={false}
```

Comment:
Hm, I see why you updated the `aria-label` but I don't see why we had those.  Screenreaders will read the button text already.  the `type="button"` prop also not needed here.  Perhaps this used to be a bootstrap Button component or something